EasyCommands/EasyCommands/Attributes.cs

```diff
@@ -24,8 +24,19 @@ public Command(params string[] names)
     [AttributeUsage(AttributeTargets.Method)]
     public class SubCommand : Command
     {
+        public bool IsDefault { get; private set; }
+
+        /// <param name="names">The aliases used to run the command</param>
+        public SubCommand(params string[] names) : base(names)
+        {
+            IsDefault = false;
+        }
+        /// <param name="isDefault">Set if this <see cref="SubCommand"/> is the default one (only one default <see cref="SubCommand"/> max is allowed)</param>
         /// <param name="names">The aliases used to run the command</param>
-        public SubCommand(params string[] names) : base(names) { }
+        public SubCommand(bool isDefault, params string[] names) : base(names)
+        {
+            IsDefault = isDefault;
+        }
```

Comment:
Can you add an override with zero arguments that acts as the default subcommand and has no names? You may also want to double check that it doesn't clash with a zero-length params array.